l2geth/rollup/l2_gas_pricer.go

```diff
@@ -0,0 +1,43 @@
+package rollup
+
+import (
+	"math"
+)
+
+type GetTargetGasPerSecond func() float64
```

Comment:
Do you imagine this being a getter for a value in the `OVM_GasPriceOracle`? Why is `GetTargetGasPerSecond` a function while `maxPercentChangePerEpoch` and `floorPrice` are not? Is the assumption that these will always be constants?